examples/server/server.cpp

```diff
@@ -541,12 +541,16 @@ struct completion_token_output {
     json to_json(bool post_sampling_probs) const {
         json probs_for_token = json::array();
         for (const auto & p : probs) {
-            std::string txt(p.txt);
+            // If the predicted token id is the same as this.tok, then we use the text_to_send instead
```

Comment:
I'm just wondering if this only happens with `tok == p.tok`, or it also affect other tokens inside `probs_for_token` list